modules/statics/src/coins.ts

```diff
@@ -946,7 +946,7 @@ export const coins = CoinMap.fromCoins([
   tronToken('trx:weth', 'Tron Wrapped Ether', 18, 'TXWkP3jLBqRGojUih1ShzNyDaN5Csnebok', UnderlyingAsset.WETH),
   algoToken(
     'algo:USDC-31566704',
-    undefined,
+    'algo:USDC-31566704',
```

Comment:
If you change alias to be the same as the coin name, this [check](https://github.com/BitGo/BitGoJS/blob/STLX-10016-fix-token-names/modules/statics/src/map.ts#L21) will fail immediately as the coin was already defined/registered in the `CoinMap` [here](https://github.com/BitGo/BitGoJS/blob/STLX-10016-fix-token-names/modules/statics/src/coins.ts#L53) and it is trying to define/register the alias one more time.